### ISSUE TITLE
Aug 12 windows update put explorer in crashloop. I`ve rolled back to 22621.1555.55.2 and its working.

### ISSUE THREAD

**Issue by McGuim** [valinet/ExplorerPatcher#1839]
Aug 12 windows update put explorer in crashloop. I`ve rolled back to 22621.1555.55.2 and its working. Just posting so if anyone get the same problem have a light.

**Comment by PandaCakes12321** [comment]
Hmm  I am on 22h2 22621.2134  and it crashes at random while  searching and or navigating. no choice too rollback  so i am stuck with it till there's a EP patch

**Comment by wdtfshi** [comment]
I am also on the same boat, as I don't have local admin privileges, I can not rollback to another version. Hope this is fixed by the time I need to use the laptop for work on monday. Also if anyone knows a way to possibly uninstall explorer patcher until this is resolved please let me know as my laptop is currently unusable

**Comment by Siderite** [comment]
For me it crashes explorer every time I open or maximize/minimize/hide an app. After three times it says that it crashed too much and disables PE, but after that everything works (except the nice PE features, obviously). Isn't this your experience?

**Comment by Azersfl** [comment]
Same issue, just stopped working today. Right clicking the taskbar causes it to crash 3x and then disables itself. Using windows 11. Have tried prior version and that also did not work.

**Comment by pyrates999** [comment]
Did you verify that symbols were downloaded after you installed the new update and rebooted?

**Comment by PandaCakes12321** [comment]
i did enabled the console and did a reinstall and it downloaded all requirements.
some other info i spotted in windows logs. that is dropped every time it crashes and reboots
Faulting application name: explorer.exe
Version: 10.0.22621.2070
Faulting module name: Windows.UI.Xaml.dll
Version: 10.0.22621.2070
Exception code: 0xc000027b

**Comment by pyrates999** [comment]
Uninstall the KB5028254.  That is a July preview update.  Only install regular updates.

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

**Comment by HughKallen** [comment]
On my wife's win11 dell laptop, today her explorer constantly was rebooting making the laptop unusable. I don't know if any new windows updates/versions were installed. There was no way to stop it. It would not finish loading windows. I could not get to the add/remove programs screen to disable explorer crasher. It was a constant slow blinking desktop/taskbar. I tried safe mode, same thing. I tried windows repair tool, no help. There was absolutely nothing I could do, so I did a reinstall of windows keeping the data files. Then I spent half a day reinstalling her programs and settings. This is a critical critical bug.
